### ISSUE TITLE
Unity crash when pressing Play

### ISSUE THREAD

**Issue by kennethlng** [google-ar/arcore-unity-sdk#260]
I followed the instructions for setting up a Unity project with the Google AR Core package and have had no issue building the example scenes (and my own scene) to iOS. However, when I am inside Unity and attempt to press Play for testing, Unity immediately crashes. 

When I disable both the CloudAnchorController and Canvas gameObjects, no crash happens. 

I've been digging around the code and the log and unfortunately I'm struggling to find what's leading to the crash. If anybody can give me pointers, that'd be great. 

I'm using Unity 2018.1.4. 

Crash Log:
--------------------------------------------------------------------------------
```
Packing sprites:
[     64 MB ]- Skipping packing - Sprite Packer (Legacy) is off.
[     64 MB ]Reloading assemblies for play mode.
Begin MonoManager ReloadAssembly
Refreshing native plugins compatible for Editor in 0.95 ms, found 5 plugins.
Preloading 2 native plugins for Editor in 1.83 ms.
Mono: successfully reloaded assembly
- Completed reload, in  2.660 seconds
Initializing Unity.PackageManager (PackageManager) v2018.1.4 for Unity v2018.1.4f1
Registering platform support modules:
Registered platform support modules in: 0.071874s.
Native extension for OSXStandalone target not found
Native extension for iOS target not found
Refreshing native plugins compatible for Editor in 0.91 ms, found 5 plugins.
Native extension for Android target not found
Load scene 'Temp/__Backupscenes/0.backup' time: 0.492064 ms 
* Assertion at loader.c:2359, condition `loc' not met

Receiving unhandled NULL exception
Obtained 48 stack frames.
#0  0x007fff6e8b9b6e in __pthread_kill
#1  0x007fff6e7c76fe in raise
#2  0x0000013d13500e in g_logv
#3  0x0000013d135181 in g_log
#4  0x0000013d0983d5 in mono_method_get_header
#5  0x0000013cf98931 in mini_method_compile
#6  0x0000013cf9a7bf in mono_jit_compile_method
#7  0x0000013cf9a2c3 in mono_jit_compile_method
#8  0x0000013d0cb200 in mono_compile_method
#9  0x0000013d00b44f in mono_magic_trampoline
#10 0x00000126b75171 in (Unknown)
#11 0x0000011efd9bb1 in  GoogleARCoreInternal.ARPrestoCallbackManager:InitializeIfNeeded () + 0x151 (0x11efd9a60 0x11efd9e8b) [0x136fc6000 - Unity Child Domain]
#12 0x0000011efd7b7c in  GoogleARCoreInternal.ARCoreAndroidLifecycleManager:CreateSession (GoogleARCore.ARCoreSession) + 0x24c (0x11efd7930 0x11efd7bdb) [0x136fc6000 - Unity Child Domain]
#13 0x0000011efd78fd in  GoogleARCore.ARCoreSession:Start () + 0x8d (0x11efd7870 0x11efd7921) [0x136fc6000 - Unity Child Domain]
#14 0x0000011be4ca1a in  (wrapper runtime-invoke) object:runtime_invoke_void__this__ (object,intptr,intptr,intptr) + 0xfa (0x11be4c920 0x11be4cb4d) [0x136fc6000 - Unity Child Domain]
#15 0x0000013cf9cebe in mono_get_runtime_build_info
#16 0x0000013d0c99f6 in mono_runtime_invoke
#17 0x0000010110d5ca in ScriptingInvocation::Invoke(ScriptingExceptionPtr*, bool)
#18 0x000001010beb21 in MonoBehaviour::InvokeMethodOrCoroutineChecked(ScriptingMethodPtr, ScriptingObjectPtr, ScriptingExceptionPtr*)
#19 0x000001010bee56 in MonoBehaviour::InvokeMethodOrCoroutineChecked(ScriptingMethodPtr, ScriptingObjectPtr)
#20 0x000001010bcbc7 in MonoBehaviour::Start()
#21 0x000001010bc759 in MonoBehaviour::DelayedStartCall(Object*, void*)
#22 0x00000100a5852f in DelayedCallManager::Update(int)
#23 0x00000100d0c906 in InitPlayerLoopCallbacks()::EarlyUpdateScriptRunDelayedStartupFrameRegistrator::Forward()
#24 0x00000100d0deae in ExecutePlayerLoop(NativePlayerLoopSystem*)
#25 0x00000100d0df1c in ExecutePlayerLoop(NativePlayerLoopSystem*)
#26 0x00000100d0e0eb in PlayerLoop()
#27 0x00000101b43102 in PlayerLoopController::UpdateScene(bool)
#28 0x00000101b43aa7 in PlayerLoopController::EnterPlayMode(bool)
#29 0x00000101b3da0b in PlayerLoopController::SetIsPlaying(bool)
#30 0x00000101b3c0e6 in Application::TickTimer()
#31 0x0000010072e4e1 in -[EditorApplication TickTimer]
#32 0x007fff485b6739 in __NSFireTimer
#33 0x007fff46459064 in __CFRUNLOOP_IS_CALLING_OUT_TO_A_TIMER_CALLBACK_FUNCTION__
#34 0x007fff46458cd7 in __CFRunLoopDoTimer
#35 0x007fff464587da in __CFRunLoopDoTimers
#36 0x007fff4644fdab in __CFRunLoopRun
#37 0x007fff4644f1a3 in CFRunLoopRunSpecific
#38 0x007fff45737d96 in RunCurrentEventLoopInMode
#39 0x007fff45737b06 in ReceiveNextEventCommon
#40 0x007fff45737884 in _BlockUntilNextEventMatchingListInModeWithFilter
#41 0x007fff439eaa73 in _DPSNextEvent
#42 0x007fff44180e34 in -[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:]
#43 0x007fff439df885 in -[NSApplication run]
#44 0x007fff439aea72 in NSApplicationMain
#45 0x0000010074d1c7 in EditorMain(int, char const**)
#46 0x0000010074d649 in main
#47 0x007fff6e769015 in start
```

**Comment by pablisho** [comment]
Hi, thanks for the report. Yes, v1.2.1 does not support Play in the Editor targeting iOS (You can do it with an Android Device and targetting Android). This crash will be fixed in the upcoming release.

**Comment by pablisho** [comment]
Hi, this issue has been fixed in [ARCore Unity SDK v1.3.0](https://github.com/google-ar/arcore-unity-sdk/releases/tag/v1.3.0). Feel free to reopen if you still experience the issue.
